src/components/HomePage/UpcomingEvents.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import TimeSlot from '../SchedulePage/TimeSlot';
+import Grid from '@material-ui/core/Grid';
+const workshopSchedule = require('../../data/WorkshopSchedule');
+
+const useStyles = makeStyles(theme => ({
+	title: {
+		fontFamily: theme.typography.fontFamily,
+		fontWeight: 'bold',
+		paddingTop: '24px',
+		paddingBottom: '16px'
+	},
+	eventsContainer: {
+		paddingTop: '16px',
+		paddingBottom: '24px'
+	}
+}));
+
+const UpcomingEvents = () => {
+	const classes = useStyles();
+	const currentTime = new Date();
+	const nextTimeSlot = workshopSchedule.reduce((closestTimeSlot, timeSlot) => {
+		if ((currentTime < timeSlot.startTime && timeSlot.startTime < closestTimeSlot.startTime) ||
+			currentTime > closestTimeSlot.startTime) {
+			return timeSlot;
+		}
+		return closestTimeSlot;
+	}, workshopSchedule[0]);
+	return <Container maxWidth='md' className={classes.eventsContainer}>
+		<Grid container justify='center' spacing={1}>
+			<Grid item md={11}>
```

Comment:
Ah, we really should try to avoid this. I believe the reason why it's necessary is because of the fact that HothDescription is split into 4/7, which doesn't add up to 12. If you change that component to be 4/8, then it'll look better.